CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing
+
+Both the [dokku ansible role](https://github.com/dokku/ansible-dokku) and [dokku](https://github.com/dokku/dokku) itself are open source projects and contributions are highly welcome!
```

Comment:
Can we reference the org's CONTRIBUTING.md?

Reply:
Sure, you mean this link https://github.com/dokku/dokku/blob/master/CONTRIBUTING.md , correct?

Reply:
Nah we have a global org thing here: https://github.com/dokku/.github